api-operator/deploy/controller-artifacts/operator.yaml

```diff
@@ -33,7 +33,8 @@ spec:
       containers:
         - name: api-operator
           # Replace this with the built image name
-          image: wso2/k8s-api-operator:1.2.0-alpha
+#          image: wso2/k8s-api-operator:1.2.0-alpha
```

Comment:
Any reason we keep this commented line? and referring non-wso2 image?

Reply:
This was included only for pre-release testing purposes. Removed in latest code.